CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGE LOG
 
+#2.7.5
+**REMOVED:** Removed Sinarmas virtual account as a payment source type
+
+#2.7.4
+**CHANGED:** Changed variable names for Alipay barcode payment
```

Comment:
I thought we have `metadata` included in `2.7.4`
Could you add below description?
```
**NEW:** Add metadata field to creating and updating a Transfer.
**NEW:** Add metadata field to creating a Refund.
**NEW:** Add metadata field to updating a Dispute.
```

Reply:
@Namiii Sorry one more 😭

Reply:
alright, will do

Reply:
👍